rpn.js

```diff
@@ -12,9 +12,13 @@ for (let i = 2; i < args.length; i++) {
         var val1 = stack.pop();
         var val2 = stack.pop();
         stack.push(val1 - val2);
+    } else if (arg === "*") {
```

Comment:
thank you for your contribution, would you mind documenting this with a comment

Reply:
sure no problem

Reply:
thank you for your comment